test.disabled/ejabberd_tests/tests/mam_helper.erl

```diff
@@ -156,6 +158,14 @@ assert_respond_query_id(_P, _ExpectedQueryId, #result_iq{query_id=not_supported}
 assert_respond_query_id(_P, ExpectedQueryId, #result_iq{query_id=QueryId}) ->
     ?assert_equal(ExpectedQueryId, QueryId).
 
+%% @doc Calls wait_archive_respond and checks "complete" field value
+wait_for_complete_respond(P, Alice, Expected) ->
```

Comment:
1. `Expected` is not very obvious on **what** is expected. Proper name would even obsolete the comment.
2. Perhaps `wait_for_complete_response`?